external/storm-kafka/src/jvm/storm/kafka/DynamicBrokersReader.java

```diff
@@ -149,4 +158,19 @@ private Broker getBrokerHost(byte[] contents) {
         }
     }
 
+    private void validateConfig(final Map conf) {
+        if(conf.get(Config.STORM_ZOOKEEPER_SESSION_TIMEOUT) == null) {
```

Comment:
@harshach there are no defaults for these values, hence the reason for this pull request. The constructor for the DynamicBrokerReader accepts a `java.util.Map` Which must contain the values that are verified in this `validateConfig` method.